### PR TITLE
docs: update usage

### PR DESCRIPTION
**Pull request by rchubarkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,60 @@ npm install framework
...
+import { createHatch, withHatch } from 'framework';
+
+import { historyPush } from 'entities/navigation';
+import { $isAuthenticated } from 'entities/session';
+
+const hatch = createHatch(createDomain())
+// domain is optional
+// for debug/logging
+
+const Page = withHatch(hatch, () => {
```
Comment (sergeysova) [inline README.md]:
```suggestion
export const Page = withHatch(hatch, () => {
```

File: README.md [hunk 2]
```diff
@@ -11,11 +11,60 @@ npm install framework
...
+
+```ts
+// some-page.ts
+import { guard, createDomain } from 'effector';
+import { createHatch, withHatch } from 'framework';
+
+import { historyPush } from 'entities/navigation';
+import { $isAuthenticated } from 'entities/session';
+
+const hatch = createHatch(createDomain())
```
Comment (sergeysova) [inline README.md]: I think it is better to move hatch to another file.

**Review by sergeysova — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rchubarkin) [reply README.md]: @sergeysova done

File: README.md (hunk 1 above)
Comment (rchubarkin) [reply README.md]: @sergeysova done

**Review by sergeysova — APPROVED** [review]

**Pull request merged**